src/parser/semantic_check.c

```diff
@@ -8317,6 +8317,12 @@ pt_check_create_entity (PARSER_CONTEXT * parser, PT_NODE * node)
 	}
     }
 
+  /* get default value of reuse_oid from system parameter, if don't use table option related reuse_oid */
```

Comment:
in line number 8220, 8228, and so on, the found_reuse_oid_option is used as initial setting value without value changing.